### DISCUSSION TITLE
0x80070570 error during installation

### DISCUSSION THREAD

**Discussion by gouravp11** [SafeExamBrowser/seb-win-refactoring#816] [Q&A]
I am trying to install the latest version of **SEB (3.6.0.633) on windows 11** and facing this error. I have already **done manual installation of .net SDK, Visual C++ 2015-2022 redistributable** but still facing the same problem.
Afterwards, I installed **previous version of SEB (3.1.1.250)** and it got **installed successfully**.

**Comment by dbuechel** [comment]
SEB also needs the WebView2 runtime as specified in the release notes or readme. But either way please post the entire log, otherwise we cannot do much to investigate the issue you're facing.
